### ISSUE TITLE
Add flag to replace batchnorm with Filter Response Normalization Layer

### ISSUE THREAD

**Issue by NikolasMarkou** [NikolasMarkou/blind_image_denoising#13]
Filter Response Normalization Layer: Eliminating Batch Dependence in the Training of Deep Neural Networks, 2020
